pkg/serverless/invocationlifecycle/trace.go

```diff
@@ -63,6 +64,9 @@ func (lp *LifecycleProcessor) startExecutionSpan(event interface{}, rawPayload [
 			inferredSpan.Span.TraceID = traceContext.TraceID
 			inferredSpan.Span.ParentID = traceContext.ParentID
 		}
+		if traceContext.TraceIDUpper64Hex != "" {
+			executionContext.TraceIDUpper64Hex = traceContext.TraceIDUpper64Hex
```

Comment:
nit.

I _think_ you do not need the conditional here. You can just always set the upper 64 from the context.

On each invocation, the execution context is reset.

When execution reaches this point, `executionContext.TraceIDUpper64Hex` will therefore _always_ be `""`.

Therefore, there is no need to avoid setting `executionContext.TraceIDUpper64Hex` to `""`.

Reply:
What's more expensive, `if traceContext.TraceIDUpper64Hex != "" ` or `executionContext.TraceIDUpper64Hex = traceContext.TraceIDUpper64Hex`?

Reply:
Not sure actually. But I just realized that we shouldn't set the `Upper64BitsTag` tag if the upper 64 is empty. So we should leave the conditional, at least for that portion.

Reply:
In fact 💡, instead of doing the del of the `Upper64BitsTag` key at end invocation, you could do it here. That would keep all logic that mutates the requestHander's meta tags in a single location.

Reply:
hm, clever